Source/Core/Common/x64Emitter.h

```diff
@@ -971,10 +971,10 @@ class XEmitter
 	// (this method might be a thunk in the case of multi-inheritance) so we
 	// have to go through a trampoline function.
 	template <typename T, typename... Args>
-	static void CallLambdaTrampoline(const std::function<T(Args...)>* f,
+	static T CallLambdaTrampoline(const std::function<T(Args...)>* f,
 	                                 Args... args)
```

Comment:
The alignment of this line (and the equivalent line in Arm64Emitter.h) is wrong now. You might as well put `Args... args)` on the first line, because it's not that long.